opm/output/eclipse/EclipseIO.cpp

```diff
@@ -62,30 +62,33 @@
 #include <sstream>
 #include <unordered_map>
 #include <utility>    // move
-
+#include <vector>
 
 namespace {
 
-inline std::string uppercase( std::string x ) {
-    std::transform( x.begin(), x.end(), x.begin(),
-        []( char c ) { return std::toupper( c ); } );
+std::string uppercase(std::string x)
```

Comment:
maybe we should just reuse this from String.hpp ?

Reply:
> maybe we should just reuse this from String.hpp ?

Yes I think that's a good idea.  I'll also fix the error&ndash;treating `FIELD` as any other group&ndash;that slipped through the refactoring of this code.